src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFOutlineService.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.hmcts.reform.em.stitching.pdf;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.destination.PDPageDestination;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.destination.PDPageFitWidthDestination;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.outline.PDDocumentOutline;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.outline.PDOutlineItem;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.em.stitching.domain.Bundle;
+
+import java.io.IOException;
+
+@Service
+public class PDFOutlineService {
+
+    private PDDocument doc;
+    private Bundle bundle;
+    private PDOutlineItem parentOutline;
+
+    PDFOutlineService() {
+    }
+
+    PDFOutlineService(PDDocument doc, Bundle bundle) {
+        this.doc = doc;
+        this.bundle = bundle;
+    }
+
+    public void createOutlines() {
+        createParentOutline();
+        int TOCPage = 0;
+
+//       // check if bundle has cover page then do a check
+//        if (bundle.hasCoversheets()) {
```

Comment:
A bundle doesn't have a coversheet - I think you're thinking of the coverpage that Tom is working on.